doc/radosgw/dynamicresharding.rst

```diff
@@ -62,98 +62,103 @@ Configuration
 .. confval:: rgw_reshard_progress_judge_interval
 .. confval:: rgw_reshard_progress_judge_ratio
 
-Admin commands
+Admin Commands
 ==============
 
-Add a bucket to the resharding queue
+Add a Bucket to the Resharding Queue
```

Comment:
Just today in a different context I was told to use sentence case for headings, but here for Ceph docs I think I like making them title case.

Reply:
Title case seems to be the majority, indeed.